### PR TITLE
#170947377 Configure storybook

### PR DESCRIPTION
**Pull request by AbdoulNuru**
#### What does this PR do?
- Configure storybook for component development.
#### Description of Task to be completed?
- install all required dependencies for storybook.
- add storybook script to package.json.
#### How should this be manually tested?
-  clone the repo and checkout to this branch ``ch-configure-storybook-170947377``
- run `npm i` to install dependencies.
- run `npm run storybook` to start the storybook.
#### What are the relevant pivotal tracker stories?
[#170947377](https://www.pivotaltracker.com/story/show/170947377)
#### Screenshots (if appropriate)
Screen Shot 2020-04-20 at 17 21 45

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    stories: ['../src/**/*.stories.[tj]s'],
+};
```
Comment (Moise1) [inline .storybook/main.js]: @AbdoulNuru , please make a new line at the of file to remove the eslint error message.

Comment (AbdoulNuru) [reply .storybook/main.js]: done @Moise1 thanks.

**Review by hakoemmy — APPROVED** [review]

**Review by Moise1 — APPROVED** [review]

**Review by blaise82 — APPROVED** [review]

**Review by AJAkimana — COMMENTED** [review]
Write a test story. For instance one of `App.js`

**Comment by AbdoulNuru** [comment]
> Write a test story. For instance one of `App.js`

@AJAkimana Thanks it's done

**Pull request merged**